### ISSUE TITLE
Include labels/probabilities in inference script outputs.

### ISSUE THREAD

**Issue by sleepinyourhat** [nyu-mll/jiant#507]
Format the output from the REPL in a way that's easier to evaluate manually: 
Show predicted labels and probabilities in addition to logits.

**Comment by zphang** [comment]
This entails adding a header, and logits and probabilities should just be different columns?

**Comment by sleepinyourhat** [comment]
Sounds good to me! Header doesn't seem strictly necessary, but definitely a good idea.
